mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -325,8 +350,13 @@ final class OneVsRest @Since("1.4.0") (
       multiclassLabeled.persist(StorageLevel.MEMORY_AND_DISK)
     }
 
+    val iters = Range(0, numClasses).par
+    iters.tasksupport = new ForkJoinTaskSupport(
+      new ForkJoinPool(Math.min(getParallelism, numClasses))
```

Comment:
Actually, quick question: Where are docs for sameThreadExecutor? ...having trouble finding API docs on Google.